code/controllers/subsystem/ticker.dm

```diff
@@ -309,15 +309,12 @@ SUBSYSTEM_DEF(ticker)
 	cinematic.mouse_opacity = MOUSE_OPACITY_TRANSPARENT
 	cinematic.screen_loc = "1,0"
 
-	var/obj/structure/bed/temp_buckle = new(src)

```

Comment:
This is the source of quite some runtimes and I myself don't see the need for buckling mobs down. Walking around doesn't break shit

Reply:
sure why not